meta-luneos/recipes-webos-ose/nyx-modules/nyx-modules/sargo.cmake

```diff
@@ -32,8 +32,3 @@ set(NYXMOD_OW_DEVICEINFO				FALSE)
 set(NYXMOD_OW_SYSTEM					FALSE)
 set(NYXMOD_OW_LED						FALSE)
 set(NYXMOD_OW_HAPTICS					FALSE)
-
-add_definitions(-DKEYPAD_INPUT_DEVICE=\"/dev/input/event1\")
-add_definitions(-DBATTERY_SYSFS_PATH=\"/sys/class/power_supply/battery/\")
-add_definitions(-DTOUCHPANEL_DEVICE=\"/dev/input/event2\")
-add_definitions(-DCHARGER_AC_SYSFS_PATH=\"/sys/class/power_supply/usb/\")
```

Comment:
These removals are a bit surprising: does is actually improve the hardware support?

Reply:
Well having them doesn't do anything to improve it by the looks of it. 
The battery and charger are on the expected path and work without it. (battery % shows, plugging and unplugging the charger cable makes the status of the charger change).

Touchpanel is on event1, so I'm not sure how it could even work on event2.

KEYPAD_INPUT_DEVICE shouldn't be needed I think? We don't have it for most other targets. It should be mainly useful for qemu from what I can tell? Other targets use nyx.conf for keys?

Reply:
Ok, well as long as the battery and charger still work, I'm fine. We can customize the other paths later on if we find it's needed.